docs/interaction.rst

```diff
@@ -22,7 +22,7 @@ example, we use more convenient names.
 
 DataArrays
 are a more human-friendly way of handling numerical arrays.
-DataArrays label the dimensions of an array provide
+DataArrays label the dimensions of an array and provide
```

Comment:
We should somehow make it clear that we're using sympl.DataArrays instead of xarray.DataArrays, or more ideally we should write wrapper code for components (or in a State object) that automatically converts any xarray.DataArray to sympl.DataArray.

Reply:
done. I'm not sure if we need the latter functionality, as long we make it clear that we are not using the base xarray DataArray. It might be a good option to add while creating a sympl DataArray though...